content/en/serverless/installation/dotnet.md

```diff
@@ -245,7 +245,7 @@ To install and configure the Datadog Serverless Plugin, follow these steps:
 {{< /tabs >}}
 
 ## What's next?
-
+- Turn on [threat monitoring][6] to get alerted on attackers targeting your service.
```

Comment:
Suggesting to move all the `What's next` section additions to somewhere after the `You can now...` or `Congratulations! You can now...` lines, as the implication is that this step is needed to view metrics, logs, and traces. Please let me know if I'm misunderstanding anything in this regard.